src/screens/Messaging/AddChatroom/AddChatroomScreen.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-console */
+import React, { useState, useEffect } from 'react'; // , {  useEffect, userReducer, useState, useReducer, }
+import { View, Text, Alert } from 'react-native';
+import { Button } from 'react-native-elements';
+import PropTypes from 'prop-types';
+import { useForm, Controller } from 'react-hook-form';
+import { TextInput } from 'react-native-gesture-handler';
+import firestore from '@react-native-firebase/firestore';
+import { styles } from './styles';
+
+// TODO: Refactor this into a new modal screen. Will require new navigation stack.
+// Using form from react-hook-form: see https://react-hook-form.com/get-started#ReactNative
+export default function AddChatroomScreen({ navigation }) {
+  const uid = navigation.getParam('uid', null);
+  const { control, handleSubmit, errors } = useForm();
+
+  function createRoom(data) {
```

Comment:
can we add descriptor to what data is? (i.e. add a comment that describes the data object, or make the name more descriptive ```roomProperties```